infoschema/infoschema.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/model"
 	"github.com/pingcap/tidb/table"
+	_ "github.com/pingcap/tidb/table/tables"
```

Comment:
It's the best way for implement and interface.

the same style as `database/sql`

```
import "database/sql"
import _ "github.com/go-sql-driver/mysql"

db, err := sql.Open("mysql", "user:password@/dbname")
```